src/utils/Utils.ts

```diff
@@ -515,6 +515,17 @@ export default class Utils {
     return 0;
   }
 
+  public static convertWattHourToKiloWattHour(wattHours: number, decimalPlaces?: number): number {
+    if (decimalPlaces) {
+      return Utils.truncTo((wattHours / 1000), decimalPlaces);
+    }
+    return Utils.convertToFloat((wattHours / 1000));
+  }
+
+  public static convertWhValuesTokWhValues(WhValues: number[], decimalPlaces?: number): number[] {
```

Comment:
No need an Utils method here, this code can be done directly outside.
To remove.

Reply:
done